src/docs/AppCanvas.vue

```diff
@@ -162,12 +174,16 @@ onMounted(() => {
           type="textarea"
           placeholder="Please input"
         />
-        <p>
+        <p v-if="preservePerWordFormatting">
           💬 To keep the demo app simple while showing the library's rich text
           features, the word "ipsum" is always rendered in italics/red without a
           stroke, and the word "consectetur" always in bold/blue with a cyan
           stroke fixed at 0.5px. Both words are also configured not to have an
           underline or a strikethrough regardless of the setting being enabled.
+          <strong
+            >This special formatting is only active when "Preserve per-word
+            formatting" is checked.</strong
+          >
```

Comment:
This formatting is inconsistent with the rest of the code here, and AFAIK, Prettier.
```suggestion
          <strong>
            This special formatting is only active when "Preserve per-word
            formatting" is checked.
          </strong>
```